pandas/core/series.py

```diff
@@ -3100,8 +3100,10 @@ def nlargest(self, n=5, keep='first'):
             When there are duplicate values that cannot all fit in a
             Series of `n` elements:
 
-            - ``first`` : take the first occurrences based on the index order
-            - ``last`` : take the last occurrences based on the index order
+            - ``first`` : return the first `n` occurrences in the
```

Comment:
Is "given index order" the correct description?

Reply:
I will include this, but when keep="last", the row labels are sorted in reverse order even in the examples in #24268 
And I feel "given index order" is correct because it is "given" by the user as input, if it is not an implicit RangeIndex

Reply:
I understand what you are writing here but I find the readability rather difficult. Any reason we don't use the same terminology in the `unique` docstring? 

`Uniques are returned in order of appearance`

^ obviously accounting for reversal with the `last` argument

Reply:
@WillAyd What do you suggest me to do? If it's changing the `unique` docstring, I want to leave this as-is, because I still want to somehow emphasize the fact that index order is reversed when `keep="last"`